security/external_secrets_operator/external-secrets-log-levels.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="external-secrets-log-levels"]
+= Customizing the External Secrets Operator for Red Hat OpenShift using the ExternalSecretsConfig custom resource
```

Comment:
I think that "using the ExternalSecretsConfig custom resource" I think is unnecessary unless you're showing more than one way of doing a task

(Same for its title in the topic map)